### PR TITLE
Update landing page (feature cards, reword, and links)

### PR DESCRIPTION
**Pull request by rocco8773**
This PR focuses on updating the hack.plasmapy.org landing page by...

1. Adding feature cards (similarly to those on the plasmapy.org) on the landing page
    - there a 3 cards: one linking to the discord channel, one linking to the 2021 Hack Week about page, and one linking to the 2021 Hack Week registration page
2. Re-worded the section for "What is a Hack Week?" to include what is hacking.  And chose language, that is hopefully, less obstructive to readers.
3. Re-worded the registration section such that it links to the 2021 Hack Week registration page.

### CODE REVIEW

**Comment by rocco8773** [comment]
@dschaffner I took what you wrote in the hackmd document...

> The term hacking has evolved from a secretive activity done by programming prodigy to just a short-hand for coding or programming. More and more programmers are spending time coding together, either collaboratively on the same project or in parallel on separate projects. Occassionally, a group will conviene an extended social coding and development session called a hackathon.

and modified it in way that hopefully kept with the essences of your original paragraph but linked up better with the following paragraph.  Let me know what you think.

**Comment by rocco8773** [comment]
For those that can build their own version of the site, this is what the landing page looks like with this PR...

image

**Comment by dschaffner** [comment]
> @dschaffner I took what you wrote in the hackmd document...
> 
> > The term hacking has evolved from a secretive activity done by programming prodigy to just a short-hand for coding or programming. More and more programmers are spending time coding together, either collaboratively on the same project or in parallel on separate projects. Occassionally, a group will conviene an extended social coding and development session called a hackathon.
> 
> and modified it in way that hopefully kept with the essences of your original paragraph but linked up better with the following paragraph. Let me know what you think.

Looks good to me.

File: pages/index.md [hunk 1]
```diff
@@ -1,20 +1,77 @@
...
 Hello world!  We are pleased to announce that the inaugural **Plasma
 Hack Week** will be held remotely from June 28 – July 2, 2021.  In the
 week before the event, we will hold prequel tutorials on topics such
 as Python, git & GitHub, contributing to an open source project, and
 writing clean scientific software.  
 
-## What is a hack week?
+## What is a hack week and hacking?
+
+The term hacking has broad range of use in modern day slang.  However,
```
Comment (svincena) [inline pages/index.md]:
```suggestion
The term hacking has a broad range of use in modern day slang.  However,
```

**Comment by rocco8773** [comment]
@dschaffner re-check the screenshot.  I accidentally took the original screenshot before updating the build.

File: pages/index.md [hunk 2]
```diff
@@ -1,20 +1,77 @@
...
 A [hack week](https://doi.org/10.1073/pnas.1717196115) is a combination
-of a hackathon and a summer school.  While hackathons have traditionally
-prioritized collaborative code development activities, a hack week
-includes a pedagogical component.  Hack weeks typically have both 
-structured learning activities such as tutorials as well as unstructured
-project time. A hack week is a chance to:
+of a hackathon (a marathon of hacking) and a summer school.  While 
+hackathons have traditionally prioritized collaborative code development
+activities, a hack week includes a teaching component.  Hack weeks 
+typically have both structured learning activities such as tutorials as
```
Comment (svincena) [inline pages/index.md]:
```suggestion
typically have both structured learning activities (such as tutorials) as
```

File: pages/index.md [hunk 3]
```diff
@@ -1,20 +1,77 @@
...
 Hack Week** will be held remotely from June 28 – July 2, 2021.  In the
 week before the event, we will hold prequel tutorials on topics such
 as Python, git & GitHub, contributing to an open source project, and
 writing clean scientific software.  
 
-## What is a hack week?
+## What is a hack week and hacking?
+
+The term hacking has a broad range of use in modern day slang.  However,
+it is often implied that hacking is done by some prodigy or expert.
```
Comment (dschaffner) [inline pages/index.md]:
```suggestion
The term hacking has a broad range of use in modern day slang, but
it is commonly believed that hacking is done by some prodigy or expert.
```

**Review by svincena — CHANGES_REQUESTED** [review]

File: pages/index.md (hunk 1 above)
Comment (rocco8773) [reply pages/index.md]: Just caught that one myself!

File: pages/index.md [hunk 4]
```diff
@@ -1,20 +1,77 @@
...
+This is not necessarily the case for the **Plasma Hack Week**.  Hacking
```
Comment (dschaffner) [inline pages/index.md]:
```suggestion
This is not the case for the **Plasma Hack Week** or scientific coding in general.  Hacking
```

Comment (rocco8773) [reply pages/index.md]: I tweaked your suggestion a little bit, so let me know what you think.

**Review by namurphy — APPROVED** [review]

**Review by smithsp — COMMENTED** [review]
Looks good to me.

**Pull request merged**